process/block/metrics.go

```diff
@@ -211,15 +211,17 @@ func indexRoundInfo(
 		Timestamp:        time.Duration(header.GetTimeStamp()),
 	}
 
-	go indexerHandler.SaveRoundInfo(roundInfo)
-
 	if check.IfNil(lastHeader) {
+		go indexerHandler.SaveRoundsInfos([]indexer.RoundInfo{roundInfo})
 		return
 	}
 
 	lastBlockRound := lastHeader.GetRound()
 	currentBlockRound := header.GetRound()
 	roundDuration := calculateRoundDuration(lastHeader.GetTimeStamp(), header.GetTimeStamp(), lastBlockRound, currentBlockRound)
+
+	roundsInfos := make([]indexer.RoundInfo, 0)
+	roundsInfos = append(roundsInfos, roundInfo)
```

Comment:
With this implementation the highest round would be first in this slice, and then the others would be sorted from lower to higher. It is ok ?
I meant that if let say currentBlockRound = 10 and lastBlockRound = 6 the results of the roundsInfos slice will be:
10, 7, 8, 9

Reply:
that's how it is but is not important order of elements in slice in this case /